src/client/src/mockData/extensionModules/mockData/mockWebPlatformData.ts

```diff
@@ -103,6 +105,8 @@ const frameworks = [
     name: "Node",
     displayName: "Node.js/Express",
     icon: "",
+    iconBase64:
```

Comment:
Do we want to put "mock" images rather than the ones that are currently on the templates?